scripts/make_port_status.py

```diff
@@ -169,6 +181,16 @@ def get_mathlib4_module_commit_info(contents):
                 mathlib4_pr=pr_info['pr'],
                 mathlib4_file=pr_info['fname'],
                 source=dict(repo=pr_info['repo'], commit=pr_info['commit']))
+
+            try:
+                pr = prs[pr_info['pr']]
+            except KeyError:
+                pass
+            else:
+                labels = [{'name': l.name, 'color': l.color} for l in pr.labels]
+                if labels:
+                    new_status.update(labels=labels)
```

Comment:
This only gives us the labels for the non-sync PRs; but that's probably fine, I don't think the labels are that useful for the sync PRs.

Reply:
Hmm. If there are more than 1 sync-PR for a file it wouldn't be canonical what labels should be displayed.

I'd say this would be easy to change if the labels actually helped you syncing

Reply:
The labels could be stored per-pr number; but I agree that this isn't particularly valuable and can be left for a future change.